recipes/libarchive/all/patches/3.5.1-enforce-conan-acl.patch

```diff
@@ -0,0 +1,224 @@
+--- CMakeLists.txt	2020-12-25 22:29:38.000000000 -0300
++++ CMakeLists_fix.txt	2021-07-13 22:19:23.435169446 -0300
+@@ -1776,218 +1776,11 @@
+ #
+ # Check for ACL libraries, headers, and functions
+ #
+-# The ACL support in libarchive is written against the POSIX1e draft,
```

Comment:
You can decrease the size of the patch by surrounding the disable code with `if(0) ... endif()`.

Reply:
Makes sense. I'll update it.
I wasn't planning on merging this tho, I made it a draft PR and linked it to the issue for the posterity.
The patch is removing a lot of tests used to set some flags. I don't have the time to study the implications of wiping all of this. 
Granted, it makes the behavior of the recipe predictable and fixes the missing acl issue, but it might be disabling some functionalities.